py3dtilers/TilesetReader/tileset_tree.py

```diff
@@ -4,32 +4,41 @@
 
 class TilesetTree(GeometryTree):
 
-    def __init__(self, tileset, tileset_paths):
+    def __init__(self, tileset, tileset_paths, geometric_errors=[None, None, None]):
         root_tile = tileset.get_root_tile()
 
         root_nodes = list()
         for i, tile in enumerate(root_tile.attributes['children']):
             offset = [c * -1 for c in tile.get_transform()[12:15]]
-            root_nodes.append(self.tile_to_node(tile, tileset_paths[i], offset))
+            root_node, depth = self.tile_to_node(tile, tileset_paths[i], offset, geometric_errors)
+            root_nodes.append(root_node)
 
         super().__init__(root_nodes)
 
-    def tile_to_node(self, tile, tileset_path, offset):
+    def tile_to_node(self, tile, tileset_path, offset, geometric_errors=[None, None, None]):
         """
-        Create a GeometryNode from a tile.
+        Create a GeometryNode and its children from a tile.
         :param tile: the tile to convert to node
         :param tileset_path: the path of the original tileset of the tile
         :param offset: the offset used to translate the features of this tile
+        :param geometric_errors: if not None, use the geometric error of the current depth to overwrite the tile geometric error
 
-        :return: a GeometryNode
+        :return: a GeometryNode, an Int
         """
-        geometric_error = tile.attributes["geometricError"]
+        children_depth = 0
+        children = list()
+        if 'children' in tile.attributes and len(tile.attributes['children']) > 0:
```

Comment:
Maybe be gentle with your future (code) readers and place here a little comment reminding the logic of geometric errors within a hierarchy ? 
Something like you did in the *Tiler/Reame.md that is
```
A tile should always have a lower geometric error than its parent. The root of the tileset should have the highest geometric error and the leaves the lowest geometric error.
```